### ISSUE TITLE
What is the BeatS lowest learning rate in Pretrain and Distill

### ISSUE THREAD

**Issue by liruixinxinxin** [microsoft/unilm#1701]
Hi Team,
I want to know the value of lowest value of learning rate in Pretrain and Distill , But the paper just supply the peak value of learning rate, can you help me, Thank you very much!

![Image](https://github.com/user-attachments/assets/f9fca62a-1335-43e4-8949-02440c244f31)
